README.md

```diff
@@ -136,10 +139,11 @@ minimally useful as early as possible. Our current planned stages are:
 Implement core functionality:
```

Comment:
Do you want to say that V1 is complete?

Reply:
It's not quite yet, there are still some things missing like only authorizing particular buyers on a given auction. I'm planning on soon going through the WICG proposal and filing an issue for each API feature that hasn't been implemented yet, with an appropriate milestone attached to each one.